.clang-format

```diff
@@ -0,0 +1,53 @@
+AccessModifierOffset: -4
+AlignTrailingComments: true
+AllowAllParametersOfDeclarationOnNextLine: true
+AllowShortFunctionsOnASingleLine: Empty
+AllowShortIfStatementsOnASingleLine: Never
```

Comment:
Let's allow short if statements on a single line -- it adds unnecessary size to our files otherwise and can make things a bit harder to read.